polaris.shopify.com/content/tools/stylelint-polaris/rules/colors-at-rule-disallowed-list.md

```diff
@@ -0,0 +1,15 @@
+---
+title: colors/at-rule-disallowed-list
+description: Disallows use of legacy color mixins
+keywords:
+  - stylelint
+  - colors
+  - colors rules
+---
+
+```diff
+// Do
++ fill: var(--p-icon-subdued);
+// Don't
+- fill: recolor-icon(--p-text-subdued);
+```
```

Comment:
Additional content is wrapped around this markdown in [rule page template](https://github.com/Shopify/polaris/pull/8027/files#diff-05926e24f38374acef7855d1a9dc6fb39ce5c7f885fb59471deb98d968fdd223)